src/sections/collection-list.liquid

```diff
@@ -8,9 +8,17 @@
 
     <a href="{% if collection == empty %}#{% else %}{{ collection.url }}{% endif %}">
       {% if collection.image != blank %}
-        {{ collection | img_url: '480x480' | img_tag: collection.title }}
+        {% include 'responsive-image' with
+          image: collection.image,
+          width: 480,
+          height: 480
+        %}
```

Comment:
@NathanPJF Is there a clean way to print values depending on a condition? I want to avoid creating another condition just to assign values to a variable.

<img width="762" alt="screen shot 2018-04-06 at 2 51 23 pm" src="https://user-images.githubusercontent.com/658169/38438957-0c4ebb62-39aa-11e8-8b95-0d65bdca0baa.png">

The above screenshot unfortunately does not work

Reply:
I found a solution. I pushed my changes.